spec/features/was_seed_viewer_spec.rb

```diff
@@ -1,11 +1,40 @@
 require 'rails_helper'
-include PURLFixtures
 
 describe 'was seed viewer public', js: true do
-  before do
+  include PURLFixtures
+  include WasSeedThumbsFixtures
+
+  before :each do
+    allow_any_instance_of(Embed::Viewer::WasSeed).to receive(:thumbs_list).and_return(get_thumbs_list_fixtures)
     stub_purl_response_with_fixture(was_seed_purl)
     visit_sandbox
     fill_in_default_sandbox_form
     click_button 'Embed'
   end
+
+  describe 'loading was-seed-viewer' do
+    it 'displays the viewer on the frame' do
+      expect(page).to have_css('.sul-embed-was-seed', count: 1, visible: true)
+      expect(page).to have_css('.sul-embed-was-thumb-item', count: 4, visible: true)
+      expect(page).to have_css('.sul-embed-was-thumb-item-div', count: 4, visible: true)
+    end
+    it 'loads the SLY slider and scroll' do
+      expect(page).to have_css('.sul-embed-thumb-slider-scroll', count: 1, visible: true)
+      sleep(5)
```

Comment:
Is this line necessary? I'm less than thrilled about including this if we don't need to?